app/views/services/slovensko_sk_api/index.html.erb

```diff
@@ -0,0 +1,126 @@
+<div class="container autoform">
+  <h1><strong>slovensko.sk API</strong></h1>
+  <p class="lead">Integrácia na slovensko.sk, prihlasovanie pomocou eID, automatizované zasielanie podaní cez <em>REST API</em>. <br><em>Docker kontajner. Open source.</em></p>
+  <hr>
+
+  <section>
+    <h2 class="text-center">Integrácie pre Váš informačný systém</h2>
+    <div class="row text-center">
+      <div class="col-md-4">
+        <i class="fa fa-cogs fa--feature"></i><br>
+        <p class="lead">Posielate veľké množstvo podaní na slovensko.sk? <em>Automatizujte cez hromadné podania.</em></p>
+      </div>
+
+      <div class="col-md-4">
+        <i class="fa fa-id-badge fa--feature"></i><br>
+        <p class="lead">Potrebujete štátom garantovanú identitu a&nbsp;informácie o&nbsp;prihlasovanej osobe? <em>Podporujeme prihlasovanie elektronickým občianskym preukazom.</em></p>
+      </div>
+
+      <div class="col-md-4">
+        <i class="fa fa-envelope-open fa--feature"></i><br>
+        <p class="lead">Potrebujete automatizovane preberať správy zo schránky na slovensko.sk? <em>Použite službu <%= link_to 'GovBox API', services_govbox_index_path %>.</em></p>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <h2></h2>
+  </section>
+
+  <section>
+    <h2>Balíky podpory</h2>
+    <p class="lead">Potrebujete pomôcť s vypracovaním integračných dokumentov? Potrebujete poradiť? <em>Objednajte si naše služby.</em></p>
+    <div class="row row-pricing">
+      <div class="col-md-4 ">
+        <div class="panel panel-pricing panel-default text-center">
+          <div class="panel-heading">
+            <h3>Komunitná verzia</h3>
+          </div>
+          <ul class="list-group">
+            <li class="list-group-item">Open Source</li>
+            <li class="list-group-item"><a href="https://hub.docker.com/r/skdigital/slovensko-sk-api" target="_blank">Docker kontajner</a></li>
+            <li class="list-group-item">Vzorové integračné dokumenty pre NASES</li>
+            <li class="list-group-item">Komunitná podpora</li>
+            <li class="list-group-item"><strong>Zadarmo</strong></li>
+            <li class="list-group-item"><a href="https://github.com/slovensko-digital/slovensko-sk-api" target="_blank" class="btn btn-default">GitHub</a></li>
+          </ul>
+        </div>
+      </div>
+      <div class="col-md-4">
+        <div class="panel panel-pricing panel-pricing--featured panel-primary text-center">
+          <div class="panel-heading text-center">
+            <h3>Biznis verzia<br><small></small></h3>
+          </div>
+          <ul class="list-group">
+            <li class="list-group-item"><strong>Všetko z Komunitnej verzie</strong></li>
+            <li class="list-group-item">Okamžitý prístup k vývojovému prostrediu na&nbsp;testovanie</li>
+            <li class="list-group-item">Vypracovanie integračných dokumentov pre&nbsp;Váš projekt na&nbsp;mieru</li>
```

Comment:
preco mame non-breaking space pred "na"? bol som v tom, ze to patri iba pred "a"

Reply:
zalomilo to len jedno slovo na konci co je skarede.